fava/core/charts.py

```diff
@@ -50,16 +50,25 @@ def _inventory_cost_or_value(inventory, date):
     return {p.units.currency: p.units.number for p in inventory}
 
 
+def _inventory_combine(inv1, inv2):
+    """Combine two inventory dicts adding numbers together -> amount dict."""
+    keys = set(inv1.keys()) | set(inv2.keys())
+    return {currency: inv1.get(currency, 0) + inv2.get(currency, 0)
+            for currency in keys}
+
+
 def _serialize_real_account(real_account, date):
+    children = [_serialize_real_account(account, date)
+                for _, account in sorted(real_account.items())]
+    balance_children = _inventory_cost_or_value(real_account.balance, date)
```

Comment:
If we use a `collections.Counter` here, this code should get much simpler

Reply:
Neat, I didn't even know that existed.